### PR TITLE
Add fetcher for destrieux 2009

### PR DESCRIPTION
**Pull request by AlexandreAbraham**
Add a fetcher for cortical atlas Destrieux et al. 2009.

Thanks to @mrahim for doing most of the work!

### CODE REVIEW

File: nilearn/datasets.py [hunk 1]
```diff
@@ -1066,6 +1066,64 @@ def fetch_smith_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
+def fetch_destrieux_2009(lateralized=True, data_dir=None, url=None,
+                         resume=True, verbose=1):
+    """Download and load the Smith ICA and BrainMap atlas (dated 2009)
```
Comment (mrahim) [inline nilearn/datasets.py]: Wrong description. (copy/paste mistake)

Comment (AlexandreAbraham) [inline nilearn/datasets.py]: Fixed.

File: nilearn/datasets.py [hunk 2]
```diff
@@ -1066,6 +1066,64 @@ def fetch_smith_2009(data_dir=None, url=None, resume=True, verbose=1):
...
+                         resume=True, verbose=1):
+    """Download and load the Destrieux cortical atlas (dated 2009)
+
+    Parameters
+    ----------
+    lateralized: boolean, optional
+        If True, returns an atlas with distinct regions for right and left
+        hemispheres.
+    data_dir: string, optional
+        Path of the data directory. Use to forec data storage in a non-
```
Comment (bthirion) [inline nilearn/datasets.py]: 'Used to force'

**Comment by banilo** [comment]
+1

**Pull request merged**